src/nativestore/PropertyEdgeLink.h

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
Lets add the standard license header here

Reply:
Done.

Reply:
Ok